### PR TITLE
Add keyring dependency

### PR DESCRIPTION
**Pull request by nallath**
CURA-7180

### CODE REVIEW

**Comment by rburema** [comment]
See also: https://github.com/Ultimaker/Cura/pull/8986

File: projects/python_packages.cmake [hunk 1]
```diff
@@ -48,7 +48,9 @@ else()
             COMMAND ${Python3_EXECUTABLE} -m pip install http://software.ultimaker.com/cura-binary-dependencies/numpy-1.15.4+mkl-cp35-cp35m-win32.whl
             COMMAND ${Python3_EXECUTABLE} -m pip install http://software.ultimaker.com/cura-binary-dependencies/scipy-1.2.0-cp35-cp35m-win32.whl
             COMMAND ${Python3_EXECUTABLE} -m pip install http://software.ultimaker.com/cura-binary-dependencies/Shapely-1.6.4.post1-cp35-cp35m-win32.whl
-            COMMENT "Install Numpy, Scipy, and Shapely"
+            # pywin32 is required to provide the keyring library with access to the Windows Credential Manager
```
Comment (rburema) [inline projects/python_packages.cmake:51]: How would this even work when it's only added to the Win32 build, not the amd64 one as well?

**Review by rburema — CHANGES_REQUESTED** [review]
I'm not sure how this worked when you tested it?

Considereing the time it takes to build the env, maybe you've added it locally somewhere.

Anyway, that command should be down below the first (not the second) endif.

Probably better to make it a separate add_custom_command as well anyway, as 'Numpy, Scipy and Shapely' are a coherent whole (math and geometry operations), and then there is a package we need for the keyring.

File: projects/python_packages.cmake (hunk 1 above)
Comment (konskarm) [reply projects/python_packages.cmake:51]: You are absolutely right. I placed it in the wrong place

**Comment by rburema** [comment]
Turns out we may not need the full pywin32 package, the one that comes with keyring, 'pywin32-ctypes' may already be enough if there's a workaround provided. 

I came to this conclusion after packaging pywin32 proved to be difficult with cx_freeze.

**Pull request merged**